test/release/examples/hello2-module.chpl

```diff
@@ -1,39 +1,46 @@
-/*  This program is conceptually very similar to hello.chpl, but it
- *  uses a more structured programming style, explicitly defining a
- *  module, a configuration constant, and a main() procedure.
+// "Production-grade" hello world
+
+/* This program is conceptually very similar to :ref:`hello.chpl
+   <primers-hello>`, but it uses a more structured programming style,
+   explicitly defining a module, a configuration constant, and a
+   main() procedure.
  */
 
 //
-// define a module named 'Hello'.  If a source file defines no
-// modules, the filename minus its .chpl extension serves as the
-// module name for the code it contains.  Thus, 'hello' would be
-// the automatic module name for hello.chpl.
+// The following statement declares a module named 'Hello'.  If a
```

Comment:
(I like treating identifiers as mathematical-esque symbols in English text and was trained to italicize those).